### PR TITLE
[WIP] update example applications

### PR DESCRIPTION
**Pull request by robertnishihara**

### CODE REVIEW

File: scripts/default_worker.py [hunk 1]
```diff
@@ -2,10 +2,6 @@
 import argparse
 import numpy as np
 
-import ray.array.remote as ra
-import ray.array.distributed as da
-import example_functions
-
```
Comment (mehrdadn) [inline scripts/default_worker.py]: Are these no longer used/necessary?

Comment (robertnishihara) [inline scripts/default_worker.py]: Yeah, if needed, these will be imported on the driver (and subsequently exported to the workers) as opposed to being imported on the workers directly.

Comment (mehrdadn) [inline scripts/default_worker.py]: Ah okay cool.

**Comment by robertnishihara** [comment]
Closing because out of date.

**Pull request closed without merging**